src/add.js

```diff
@@ -0,0 +1,16 @@
+const todoUL = document.querySelector('#todoUl');
+todoUL.innerHTML = '';
+export default function add(counter, todo, bool) {
+  const todoListString = localStorage.getItem('todoList');
+  const todoList = JSON.parse(todoListString);
+  const li = {
+    description: todo,
```

Comment:
- [ ] First, please make sure the app runs properly when running the command `npm i`. Your app does not add, remove or edit. Currently, I have the below situation 👇 
![ezgif com-gif-maker](https://user-images.githubusercontent.com/67087664/155527566-9232f76b-164a-4d59-8d52-ba1845da1db3.gif)
Kindly fix the issues so todos can be added and removed, edited 🙏